### ISSUE TITLE
Virtualization VS. Emulation differences in approach?

### ISSUE THREAD

**Issue by ghost** [ghaststeam/PS4EMX#15]
I notice people have suggested that virtualization is better since the PS4 OS could run on the same processor architecture as PCs. But wouldn't that not work because the GPU and other hardware will still need to be emulated? And would emulating the CPU in software not be very slow? If you combine that with the emulated GPU wouldn't it require too many resources to function even on the best processors?
I have also heard that virtualizing is harder to implement than emulation ... I tried reading comparisons but the difference between "process virtual machine" and "emulation" in general seems tricky to fully understand, specifically because process virtual machines are also emulators of some sort as well.

In short, what are these differences and how can PS4EMX utilize them?
